Automattic/wp-calypso
# Jetpack License Portal: Introduce a uniform radio field component

#### Changes proposed in this Pull Request

* This PR introduces a new component to replace the various radio components we are currently using in the Licensing Portal. It also retrofits the components and pages on the license issue page, as well as the license assignment page.

#### Testing instructions

* Check that you are able to issue a license and the radio boxes work as expected.
* Check that you are able to assign a license to a site and the radio boxes work as expected.
